### ISSUE TITLE
Missing semicolons in c++ codegen

### ISSUE THREAD

**Issue by bebidek** [wxGlade/wxGlade#495]
When creating dialog and generating C++ code, these two lines don't have semicolons:
```
SetAffirmativeId(button_OK->GetId())
SetEscapeId(button_CANCEL->GetId())
```

**Comment by DietmarSchwertberger** [comment]
Thanks for reporting. The fix is now in the repository, in both branches.

**Comment by DietmarSchwertberger** [comment]
Released v1.0.3
